src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -241,31 +262,31 @@ CrashInfo::InitializeDAC()
         m_hdac = LoadLibraryA(dacPath.c_str());
         if (m_hdac == nullptr)
         {
-            printf_error("LoadLibraryA(%s) FAILED %d\n", dacPath.c_str(), GetLastError());
+            printf_error("InitializeDAC: LoadLibraryA(%s) FAILED %s\n", dacPath.c_str(), GetLastErrorString().c_str());
```

Comment:
Can we include the error code still like the HR based ones below?

Reply:
GetLastErrorString() actually puts last error code in the string it returns.